Updates to Filter field content (#3532)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/openidconnect/appidpinokta.md

    You can enter an expression to reformat the value, if desired. For example, if the social username is `john.doe@mycompany.com`, then you could specify the replacement of `mycompany` with `endpointA.mycompany` to make the transformed username `john.doe@endpointA.mycompany.com`. See [Okta Expression Language](/docs/reference/okta-expression-language/).

> * **Filter > Only allow usernames that match defined RegEx Pattern**: Select this option to only authenticate users with transformed usernames that match a regular expression pattern in the text field that appears. This filters the IdP username to prevent the IdP from authenticating unintended or privileged users. Users are only authenticated if the transformed username matches the regular expression pattern.

Reviewer: Same question as above: "... unintended or privileged users"?

> * **Filter > Only allow usernames that match defined RegEx Pattern**: Select this option to only authenticate users with transformed usernames that match a regular expression pattern in the text field that appears. This filters the IdP username to prevent the IdP from authenticating unintended or privileged users. Users are only authenticated if the transformed username matches the regular expression pattern.

Author: ```suggestion
* **Filter > Only allow usernames that match defined RegEx Pattern**: Select this option to only authenticate users with transformed usernames that match a regular expression pattern in the text field that appears. This filters the IdP username to prevent the IdP from authenticating unintended users. Users are only authenticated if the transformed username matches the regular expression pattern.
```

>     > **Note:** When you use Okta for B2B or multi-tenancy use cases, we recommend that you select this checkbox. This helps you scope a subset of users in the org and enforce identifier constraints, such as email suffixes.

Reviewer: ```suggestion
    > **Note:** When you use Okta for B2B or multi-tenancy use cases, select this checkbox to help you scope a subset of users in the org and enforce identifier constraints, such as email suffixes.
```

Author: I made two sentences because it was longer than 25 words and becoming unwieldy.
